src/rules/pyupgrade/rules/import_replacements_six.rs

```diff
@@ -0,0 +1,200 @@
+use std::collections::HashMap;
+
+use once_cell::sync::Lazy;
+use rustpython_ast::{AliasData, Located, Stmt};
+
+use crate::ast::types::Range;
+use crate::checkers::ast::Checker;
+use crate::fix::Fix;
+use crate::registry::{Diagnostic, Rule};
+use crate::rules::pyupgrade::helpers::{get_fromimport_str, ImportFormatting};
+use crate::source_code::Locator;
+use crate::violations;
+
+static REPLACE_MODS: Lazy<HashMap<&str, &str>> = Lazy::new(|| {
+    let mut m = HashMap::new();
+    m.insert("BaseHTTPServer", "http.server");
```

Comment:
Would initialising from an array not be better suited here?
(Should be more performant, but might be compiled away anyway)
```
let solar_distance = HashMap::from([
    ("Mercury", 0.4),
    ("Venus", 0.7),
    ("Earth", 1.0),
    ("Mars", 1.5),
]);
```